config/application.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'boot'
+
+require "rails"
+require "active_model/railtie"
+require "active_job/railtie"
+require "active_record/railtie"
+require "active_storage/engine"
+require "action_controller/railtie"
+require "action_mailer/railtie"
+require "action_view/railtie"
+require "action_cable/engine"
+require "sprockets/railtie"
+Bundler.require(*Rails.groups)
+
+module FoodHub
+  class Application < Rails::Application
+    config.load_defaults 5.2
+
+    config.generators do |generate|
+      generate.helper false
+      generate.assets false
+      generate.view_specs false
+      generate.helper_specs false
+      generate.routing_specs false
+      generate.controller_specs false
+      generate.system_tests false
+    config.generators.system_tests = nil
```

Comment:
IF you choose to keep this line (it is redundant) please place it outside the current do — end block.

Reply:
Please confirm which line you are referring to, is 'config.generators.system_test =nil' ?

Reply:
Line 27. The one right above the comment. That’s the standard